content/doc/administration/requirements/java.adoc

```diff
@@ -58,3 +58,18 @@ Modern versions of Jenkins controllers and Jenkins agents verify Java requiremen
 and notify users when they are launched with an unsupported version.
 
 The plugin:versioncolumn[Versions Node Monitors plugin] provides detailed Java version monitoring.
+
+== Java Development Kits Used
```

Comment:
Since the other entries in the file use `##` to identify headings, this should be the same
```suggestion
## Java Development Kits Used
```